### PR TITLE
Implement beta wrapper component into personalization features

### PR DESCRIPTION
**Pull request by ncksllvn**
Super barebones wrapper component so that the user has to explicitly enroll in the beta to gain access. Doing it this way we shouldn't need to create a separate route for registering users - just share the `/profile-beta` link and make sure that landing component is wrapped in the `<BetaApp>`. For now, it's saving the beta information in the session storage but will need to be adjusted based on what comes out of department-of-veterans-affairs/vets.gov-team/issues/9155.

image

Resolves department-of-veterans-affairs/vets.gov-team/issues/9097

### CODE REVIEW

**Comment by ncksllvn** [comment]
I have the beta work in a separate [branch](https://github.com/department-of-veterans-affairs/vets-website/tree/beta-component), but I think it's probably way too soon to consider adding any of that work to master.

File: src/js/personalization/user-profile-beta/containers/UserProfileApp.jsx [hunk 1]
```diff
@@ -34,23 +35,25 @@ class UserProfileApp extends React.Component {
...
-                  modal={{
-                    open: this.props.openModal,
-                    currentlyOpen: this.props.profile.modal,
-                    pendingSaves: this.props.profile.pendingSaves,
-                    errors: this.props.profile.errors
-                  }}
-                  updateActions={this.props.updateActions}
-                  fetchExtendedProfile={this.props.fetchExtendedProfile}/>
+          <BetaApp featureName={features.personalization}>
+            <DowntimeNotification appTitle="user profile page" dependencies={[services.mvi, services.emis]}>
```
Comment (ncksllvn) [inline src/js/personalization/user-profile-beta/containers/UserProfileApp.jsx]: Here's where the beta stuff comes into play for our work.

**Comment by bshyong** [comment]
@ncksllvn LGTM; note that we may not need the session storage work because beta access products will be included in the available services once the API work in https://github.com/department-of-veterans-affairs/vets.gov-team/issues/9155 is complete

**Comment by ncksllvn** [comment]
@bshyong Definitely, gonna make adjustments here soon

**Pull request merged**